deployment/Makefile

```diff
@@ -1,16 +1,21 @@
-PROJECT_ID := gep-benin
-
 SHELL := /bin/bash
-COMPOSE := -f docker-compose-gep.yml -f docker-compose.yml
```

Comment:
In the latest docker compose, Project ID and Compose files are explicitly defined in the .env file.